### PR TITLE
[FEATURE] Javax bean validation

### PR DESCRIPTION
**Pull request by jscamara**
Feature to make easier to validate beans using javax annotation

### CODE REVIEW

File: heimdall-gateway/pom.xml [hunk 1]
```diff
@@ -115,6 +115,29 @@
...
+		</dependency>
+		
+		<dependency>
+		    <groupId>javax.el</groupId>
+		    <artifactId>javax.el-api</artifactId>
+		    <version>3.0.0</version>
+		</dependency>
+		 
+		<dependency>
+		    <groupId>org.glassfish.web</groupId>
```
Comment (MAAARKIN) [inline heimdall-gateway/pom.xml]: this dependency it's really necessary?

File: heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImpl.java [hunk 2]
```diff
@@ -100,21 +109,51 @@ public String parse(String string) {
 		try {
 			@SuppressWarnings("unchecked")
 			T obj = (T) mapper().readValue(json, classType);
+			
+			Set<ConstraintViolation<T>> violations = validador().validate(obj);
+
+			if (!violations.isEmpty()) {
+			     
+			     String violacoes = parse(violations.stream().map(v -> {
```
Comment (MAAARKIN) [inline heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImpl.java]: Plz, avoid portuguese words, fix this.

File: heimdall-gateway/src/test/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImplTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import br.com.conductor.heimdall.core.dto.BeanValidationErrorDTO;
+import br.com.conductor.heimdall.middleware.exception.BeanValidationException;
+import lombok.Data;
+
+@RunWith(MockitoJUnitRunner.class)
+public class JsonImplTest {
+
+     @InjectMocks
+     private JsonImpl sujeito;
```
Comment (MAAARKIN) [inline heimdall-gateway/src/test/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImplTest.java]: avoid pt-br words here too.

File: heimdall-gateway/src/test/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImplTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+     
+     @Test(expected = BeanValidationException.class)
+     public void validarAnotacaoNotBlank() {
```
Comment (MAAARKIN) [inline heimdall-gateway/src/test/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImplTest.java]: translate this name function to english plz.

Comment (MAAARKIN) [reply heimdall-gateway/src/test/java/br/com/conductor/heimdall/gateway/filter/helper/JsonImplTest.java]: and others too.

File: heimdall-gateway/pom.xml (hunk 1 above)
Comment (jscamara) [reply heimdall-gateway/pom.xml]: Hey @MAAARKIN I did some tests and you are right! Apparently the tomcat embedded provide the implementation of variable interpolation and expression.

**Pull request merged**